sql/08_create_taxonomic_table.sql

```diff
@@ -0,0 +1,69 @@
+-- Note: FIRST: edit %db_owner% to the account name of the DB owner 
+
+CREATE TABLE lter_metabase."TaxaList" (
+	"TaxonID" character varying(50) NOT NULL;
+	"TaxonomicAuthority" character varying(50) NOT NULL;
+	"TaxonName" character varying(200);
```

Comment:
the eml element names are taxonomicClassification taxonRankName taxonRankValue. And soon we will be able to also have the id. (@mob pls confirm we get both id and system, and these are going to be attributes of which element in eml2.2? Or I can look it up.) I want to use a consistent name mapping between column names of views and eml elements and attributes. I realize the names are long. But taxonname could be mistaken for a taxonomicRankValue without the Rank in there. A taxonRankName is one of {kingdom, phylum, class, order, family, genus, ...}, that is, the name of the rank. I want to keep rank in there not just for that clarity, but to stick to the pattern. I do not know if the R code follows that pattern. 
Aside from just good practice, it also avoids future name collisions.

Reply:
There's been no R code written for taxonomy, so it's a blank canvas for now. Whenever we have the SQL view down, the R code will follow that.